lib/rules/function-linear-gradient-no-nonstandard-direction/__tests__/index.js

```diff
@@ -97,5 +124,32 @@ testRule(rule, {
   }, {
     code: ".foo { background: -o-linear-gradient(to bottom, #fff, #000 ); }",
     message: messages.rejected,
+  }, {
+    code: ".foo { background: url(foo.png), -webkit-linear-gradient(to bottom, #fff, #000 ); }",
+    message: messages.rejected,
```

Comment:
Please add `line` and `column` properties to all the `reject` tests - [see here](https://github.com/stylelint/stylelint/blob/master/lib/rules/at-rule-blacklist/__tests__/index.js#L25-L27) as an example.